app/src/main/res/values-fr/strings.xml

```diff
@@ -243,11 +243,11 @@
     <string name="theme_amoled">Sombre AMOLED</string>
     <string name="theme_semi_dark">Semi-transparent sombre</string>
     <string name="theme_semi_light">Semi-transparent</string>
-    <string name="double_tap_to_lock">Double-tap pour vérouiller l\'écran</string>
-    <string name="enable_double_tap_to_lock">Pour activer le verrouillage par double tap, vous devrez accorder des permissions supplémentaires : activez KISS dans l\'écran suivant.</string>
+    <string name="double_tap_to_lock">Appui double pour verrouiller l\'écran</string>
```

Comment:
"Tap" is not French. There were already occurrences of "appui" in the file, so I suggest that.
For "verrouiller", see here: https://www.dictionnaire-academie.fr/article/A8V0343